specification/sbi_atee.adoc

```diff
@@ -1478,15 +1478,19 @@ one or more page mappings to the G-stage translation structure for a TVM.
 The pages to be used for the G-stage page table structures must have been converted
 (and tracked) by the TSM as TEE pages; otherwise this operation will not succeed.                                                                     
 
-| <<sbi_tee_host_add_tvm_measured_pages, sbi_tee_host_add_tvm_measured_pages>> | Add 
-a page for an existing mapping for a TVM page - this add_pre must be performed before 
-finalization of the TVM measurement via sbi_tee_host_finalize_tvm. For this 
-operation, the VMM must provide the page contents that get copied into 
-confidential memory pages for the TVM (and get tracked, encrypted etc). The 
-contents of these pages are also measured via the sbi_tee_host_add_tvm_measured_pages, 
-including the GPA at which the page is mapped. After the TVM msmt is 
-finalized via sbi_tee_host_finalize_tvm, no more pre-add pages are allowed by 
-the TSM for that TVM.
+| <<sbi_tee_host_add_tvm_measured_pages, sbi_tee_host_add_tvm_measured_pages>> | Copies the
+given number of pages from non-confidential memory at `source_address` to confidential
+memory at `dest_addr`, then measures and maps the pages at `dest_addr` at the TVM physical
+address space at `tvm_guest_gpa`. The mapping must lie within a region of confidential memory
+created with `sbi_tee_host_add_tvm_memory_region()`. This call must not be made after calling
+`sbi_tee_host_finalize_tvm()`.
+
+This operation is used to extend the static measurement for a TVM for added page contents.
+The operation performs a SHA384 hash extend to the measurement register managed 
+by the TSM on a 256 byte block of the page. The GPA of the page mapped is part of the 
+measurement operation. The measurement process is a state machine that must be faithfully
```

Comment:
Consider rephrasing as: The VMM must faithfully reproduce the state machine for the measurement process. Otherwise...

Reply:
AFAIK what this statement is getting at is that the order in which measured pages are added to the TVM affects the attestation evidence, and thus the VMM must faithfully preserve that order. If so, probably best to be explicit.

Reply:
Fixed.